src/JustEat.StatsD.Tests/WhenTheTransportThrows.cs

```diff
@@ -1,66 +1,81 @@
 using System;
 using System.Net.Sockets;
+using JustEat.StatsD.Buffered;
 using Shouldly;
 using Xunit;
 
+#pragma warning disable xUnit1026 // Theory methods should use all of their parameters disabled to render test case name
```

Comment:
Can you provide a screenshot of the before and after with this warning disabled?